gnomad/utils/file_utils.py

```diff
@@ -222,3 +222,57 @@ def repartition_for_join(
             " partitions than the original HT!"
         )
     return ht._calculate_new_partitions(ht.n_partitions() * new_partition_percent)
+
+
+def create_vds(
+    gvcfs: str,
+    output_path: str,
+    temp_path: str,
+    save_path: Optional[str] = None,
+    use_genome_default_intervals: bool = False,
+    use_exome_default_intervals: bool = False,
+    intervals: Optional[List[str]] = None,
+    gvcf_batch_size: Optional[int] = None,
+) -> hl.vds.VariantDataset:
+    """
+    Combine GVCFs into a single VDS.
+
+    :param gvcfs: Path to file containing GVCF paths with no header.
+    :param output_path: Path to write output VDS.
+    :param temp_path: Path to write temporary files. A bucket with a life-cycle
+        policy is recommended.
+    :param save_path: Path to write combiner to on failure. Can be used to restart
+        combiner from a failed state. If not specified, defaults to temp_path +
+        combiner_plan.json.
+    :param use_genome_default_intervals: Use the default genome intervals.
+    :param use_exome_default_intervals: Use the default exome intervals.
+    :param intervals: Path to text file with intervals to use for VDS creation.
+    :param gvcf_batch_size: Number of GVCFs to combine into a Variant Dataset at once.
+    :return: Combined VDS.
+    """
+    if not save_path and temp_path:
+        save_path = temp_path + "combiner_plan.json"
+
+    gvcfs = read_list_data(gvcfs)
+    intervals = read_list_data(intervals) if intervals else None
```

Comment:
can replace to use hl.import_locus_intervals and then collect the intervals here

Reply:
Thank you!